testing/cli-e2etest/testscenarios/config/list_config_test.go

```diff
@@ -90,13 +90,14 @@ func TestListConfig(t *testing.T) {
 		result := tracetestcli.Exec(t, "list config --sortBy name --output json", tracetestcli.WithCLIConfig(cliConfig))
 		helpers.RequireExitCodeEqual(t, result, 0)
 
-		configYAML := helpers.UnmarshalJSON[[]types.ConfigResource](t, result.StdOut)
-
-		require.Len(configYAML, 1)
-		require.Equal("Config", configYAML[0].Type)
-		require.Equal("current", configYAML[0].Spec.ID)
-		require.Equal("Config", configYAML[0].Spec.Name)
-		require.False(configYAML[0].Spec.AnalyticsEnabled)
+		configList := helpers.UnmarshalJSON[types.ResourceList[types.ConfigResource]](t, result.StdOut)
```

Comment:
this scenario was incorrect (as well as the old implementation). it was returning an array of resources instead of a `ResourceList`